index.html

```diff
@@ -92,8 +92,9 @@ <h2>package.json<span class="material-icons"></span></h2>
         Enter NPM module name here <i class="material-icons">arrow_upward</i> to see the dependency graph.  To graph multiple modules separate names with commas (e.g. <a href="?q=mocha, chalk, rimraf">"mocha, chalk, rimraf"</a>).
         </p>
         <div id="drop_target" style="text-align: center">
-					... or drop a <code>package.json</code> file here
-				</div>
+          ... or drop a <code>package.json</code> file here
+        </div>
+        <input id="file_upload" type="file" accept="application/json" style="position: fixed; top: -100em" />
```

Comment:
The `-100em` is from [here](https://stackoverflow.com/a/28075416/8887313) on Stack Overflow. Apparently, using `display: none` may cause the input to be disabled in some browsers. (Best not to have things potentially break in the future.)

Reply:
Also, comment in code (w/ link to SO article)